R/scripts/build_US_setup.R

```diff
@@ -62,20 +142,31 @@ t_commute_table <- tibble(
 )
 
 rc <- padding_table %>% bind_rows(commute_data) %>% bind_rows(t_commute_table)
+
 if(opt$w){
-  rc <- rc %>%pivot_wider(OFIPS,names_from=DFIPS,values_from=FLOW, values_fill=c("FLOW"=0),values_fn = list(FLOW=sum))
+  mobility_file <- 'mobility.txt'
+} else if (length(config$spatial_setup$mobility) > 0) {
```

Comment:
So the behavior here will be that the config filename (if it ends with .txt) overwrites the -w option in the config call

Reply:
I guess this will be good for backward compatibility